Autopilot/Path Manager/PathManager.c

```diff
@@ -340,7 +340,23 @@ char followWaypoints(PathData* currentWaypoint, float* position, float heading,
 
             *sp_Heading = (int)followStraightPath((float*)&waypointDirection, (float*)targetCoordinates, (float*)position, heading);
         }
-        else{
+        else{            
```

Comment:
On the right track. So if you look at the logic and compare it to the history of the file: https://github.com/UWARG/PICpilot/blob/54101f9019d5a16ccbd7ea7f513a9ab8b36f3a31/Autopilot/Path%20Manager/PathManager.c

You will notice a problem: This else statement gets called at every turn the aircraft makes. You now put a if statement around what was previously there. In consequence, the plane will never ever get past the first turn/waypoint it makes.

The way the autopilot works is that it alternates between PATH and ORBIT. This "else" statement is the ORBIT.

What you probably need to do is create an if, else statement for this section of code.